### ISSUE TITLE
Assert causing crash with invalid instruction bytes

### ISSUE THREAD

**Issue by mrexodia** [zyantific/zydis#49]
Hello,

Today someone reported a bug to me where x64dbg was crashing and it turns out there is an assert in Zydis that causes this. ODA gives this as an invalid instruction, but it might just be too new for it...

To reproduce I use ZydisInfo. On `master`:

```
>zydisinfo -64 62B1F80D1831
Assertion failed: instruction->operandCount >= 2, file c:\codeblocks\zydis\src\decoder.c, line 2006
```

On `develop`:
```
>zydisinfo -64 62B1F80D1831
== [    BASIC ] ============================================================================================
   MNEMONIC: vprefetche1 [ENC: MVEX, MAP: 0F, OPC: 18]
     LENGTH:  6
        SSZ: 64
       EOSZ: 64
       EASZ: 64
   CATEGORY: PREFETCH
    ISA-SET: KNCE
    ISA-EXT: KNCE
 EXCEPTIONS: NONE
 ATTRIBUTES: HAS_MODRM HAS_MVEX

== [ OPERANDS ] ============================================================================================
##       TYPE  VISIBILITY  ACTION      ENCODING   SIZE  NELEM  ELEMSZ  ELEMTYPE                        VALUE
--  ---------  ----------  ------  ------------   ----  -----  ------  --------  ---------------------------
 0     MEMORY    EXPLICIT       R      MODRM_RM      8      0      64       INT  TYPE  =                 MEM
                                                                                 SEG   =                  ds
                                                                                 BASE  =                 rcx
                                                                                 INDEX =                none
                                                                                 SCALE =                   0
                                                                                 DISP  =  0x0000000000000000
--  ---------  ----------  ------  ------------   ----  -----  ------  --------  ---------------------------

== [      AVX ] ============================================================================================
  VECTORLEN: 512
  BROADCAST: NONE
   ROUNDING: DEFAULT
        SAE: N
       MASK: k5 [MERGE]
         EH: N
    SWIZZLE: NONE
    CONVERT: NONE

== [   DISASM ] ============================================================================================
  vprefetche1 byte ptr ds:[rcx]
```

On `future`:

```
>zydisinfo -64 62B1F80D1831
== [    BASIC ] ============================================================================================
   MNEMONIC: vprefetche1 [ENC: MVEX, MAP: 0F, OPC: 18]
     LENGTH:  6
        SSZ: 64
       EOSZ: 64
       EASZ: 64
   CATEGORY: PREFETCH
    ISA-SET: KNCE
    ISA-EXT: KNCE
 EXCEPTIONS: NONE
 ATTRIBUTES: HAS_MODRM HAS_MVEX

== [ OPERANDS ] ============================================================================================
##       TYPE  VISIBILITY  ACTION      ENCODING   SIZE  NELEM  ELEMSZ  ELEMTYPE                        VALUE
--  ---------  ----------  ------  ------------   ----  -----  ------  --------  ---------------------------
 0     MEMORY    EXPLICIT       R      MODRM_RM      8      0      64       INT  TYPE  =                 MEM
                                                                                 SEG   =                  ds
                                                                                 BASE  =                 rcx
                                                                                 INDEX =                none
                                                                                 SCALE =                   0
                                                                                 DISP  =  0x0000000000000000
--  ---------  ----------  ------  ------------   ----  -----  ------  --------  ---------------------------

== [      AVX ] ============================================================================================
  VECTORLEN: 512
  BROADCAST: NONE
   ROUNDING: DEFAULT
        SAE: N
       MASK: k5 [MERGE]
         EH: N
    SWIZZLE: NONE
    CONVERT: NONE

== [   DISASM ] ============================================================================================
  vprefetche1 byte ptr ds:[rcx]
```

**Comment by mrexodia** [comment]
Oh I just saw that this was an invalid assert condition, so it appears to be fixed now.

**Comment by flobernd** [comment]
Hi there, yes this assert condition got fixed in 1aa5e648f177b05e522ecce088bf400f78015f8c (develop) and e6dbba0bd314680dda64ef34e56cd12046ae8a2d (future).

We will update the master branch in the next few days (maybe already today/tomorrow).
